src/main/java/hudson/plugins/git/extensions/impl/SparseCheckoutPath.java

```diff
@@ -3,22 +3,25 @@
 import com.google.common.base.Function;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.EnvVars;
 import hudson.Extension;
 import hudson.model.AbstractDescribableImpl;
 import hudson.model.Descriptor;
 import jenkins.model.Jenkins;
 import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import javax.annotation.Nullable;
 import java.io.Serializable;
 import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class SparseCheckoutPath extends AbstractDescribableImpl<SparseCheckoutPath> implements Serializable {
 
     private static final long serialVersionUID = -6177158367915899356L;
 
-    @SuppressFBWarnings(value="SE_TRANSIENT_FIELD_NOT_RESTORED", justification="Default value is OK in deserialization")
-    public static final transient SparseCheckoutPathToPath SPARSE_CHECKOUT_PATH_TO_PATH = new SparseCheckoutPathToPath();
```

Comment:
Won't this change break compatibility for git plugin API consumers that refer to this public field that is being deleted?

It is included in the [Javadoc](https://javadoc.jenkins-ci.org/plugin/git/hudson/plugins/git/extensions/impl/SparseCheckoutPath.html#SPARSE_CHECKOUT_PATH_TO_PATH) of the current release.

Reply:
Updated to keep this variable for API consumers.